store/iavl/wire.go

```diff
@@ -0,0 +1,7 @@
+package iavl
```

Comment:
Should we use a passed-in codec instead perhaps? Why do we need a singleton?

Reply:
`iavl.Store` needs to serialize the subspace query result, but it does not hold nor take a `cdc`